cli.js

```diff
@@ -21,6 +21,10 @@ if (!args.dir || (!args.all && (!args.platform || !args.arch))) {
   process.exit(1)
 }
 
+if (args.tmpdir === 'false') {
+  args.tmpdir = false
+}
```

Comment:
I already commented on this before; I suspect using the boolean flag is unsafe because tmpdir *is* normally a string (the location of the staging area); it is being overloaded basically to have false treated as a special value so that the staging area can be disabled.